### ISSUE TITLE
Provide some bittorrent status to users?

### ISSUE THREAD

**Issue by petterreinholdtsen** [johang/vlc-bittorrent#26]
Is it possible to provide some bittorrent status information to the user during playback?  For example the number of peers/seeds, download speed, etc.  It would be useful for slow starting torrents, to know if they hang or are just slow.

**Comment by tiotrom** [comment]
Agree. Also a progress bar of some sort....

**Comment by johang** [comment]
VLC doesn't provide a good interface for modules/plugins to do this.

**Comment by tiotrom** [comment]
I see...

**Comment by petterreinholdtsen** [comment]
Is 5d8045892c0aedafb7eebc4e6e1b18a1b3ddbefd somehow related to this?

**Comment by johang** [comment]
Yes, it is. But I didn't find a good way of providing this info to users. Any suggestions? I guess I could periodically print to the the Messages window.

I added a progress dialog downloading metadata in 0c62656.

**Comment by petterreinholdtsen** [comment]
[Johan Gunnarsson]
> Yes, it is. But I didn't find a good way of providing this info to
> users. Any suggestions? I guess I could periodically print to the the
> Messages window.

I tried asking on the #videolan IRC channel, but got no response.
Perhaps it is still a hard problem...

> I added a progress dialog downloading metadata in 0c62656.

I look forward to having the feature in Debian, after the next stable
release. :)

-- 
Happy hacking
Petter Reinholdtsen
